cli/src/commands/jobs.rs

```diff
@@ -114,16 +120,22 @@ pub async fn handle_history(api: &mut PhylumApi, matches: &clap::ArgMatches) ->
         let job_id = resolve_job_id(matches.value_of("JOB_ID").expect("No job id found"))?;
         action = get_job_status(api, &job_id, verbose, pretty_print, display_filter).await;
     } else {
-        let resp = api.get_status().await;
-        if let Err(Some(StatusCode::NOT_FOUND)) = resp.as_ref().map_err(|e| e.status()) {
-            print_user_warning!(
-                "No results found. Submit a lockfile for processing:\n\n\t{}\n",
-                Blue.paint("phylum analyze <lock_file>")
-            );
-        } else {
-            println!("Projects and most recent runs\n",);
+        if let Some(project) = matches.value_of("project") {
+            let resp = api.get_project_details(project).await.map(|r| r.jobs);
             print_response(&resp, pretty_print, None);
-        }
+        } else {
+            let resp = api.get_status().await;
+
+            if let Err(Some(StatusCode::NOT_FOUND)) = resp.as_ref().map_err(|e| e.status()) {
+                print_user_warning!(
+                    "No results found. Submit a lockfile for processing:\n\n\t{}\n",
+                    Blue.paint("phylum analyze <lock_file>")
+                );
+            } else {
+                println!("Projects and most recent runs\n",);
```

Comment:
Not really related to this issue, but this should be wrapped in `if pretty_print`
```suggestion
                if pretty_print {
                    println!("Projects and most recent runs\n",);
                }
```